packages/flame_forge2d/lib/body_component.dart

```diff
@@ -58,22 +58,18 @@ abstract class BodyComponent<T extends Forge2DGame> extends BaseComponent
         _transform.m24 != body.position.y ||
         _lastAngle != angle) {
       _transform.setIdentity();
-      _transform.scale(1.0, -1.0);
-      _transform.translate2(body.position);
-      _transform.rotateZ(angle);
+      _transform.translate(body.position.x, -body.position.y);
+      _transform.rotateZ(-angle);
       _lastAngle = angle;
     }
-    canvas.transform(_transform.storage);
-  }
 
-  @override
-  void postRender(Canvas canvas) {
-    canvas.transform(_flipYTransform.storage);
+    canvas.transform(_transform.storage);
```

Comment:
This should be removed I think?

Reply:
It is removed actually, git is messing with you :D check source

Reply:
There is only one commit, and it exists in that commit?

Reply:
![image](https://user-images.githubusercontent.com/11612804/129805032-e4d6837d-ef2c-4238-b065-a6fe7eb926f7.png)

this is how it looks

Reply:
Weeeird, never seen github do that before.

Reply:
Aaah, now I understand what the diff is showing! I must have been very tired when reviewing this yesterday. Missed that it was red lines all the way into the render method.